client/coral-embed-stream/src/graphql/index.js

```diff
@@ -129,7 +134,19 @@ export default {
             user: {
               __typename: 'User',
               id: auth.user.id,
-              username: auth.user.username
+              username: auth.user.username,
+              tags: tags.map((tag) => ({
+                tag: {
+                  name: tag,
+                  created_at: new Date().toISOString(),
+                  __typename: 'Tag'
+                },
+                assigned_by: {
```

Comment:
This field I think was not requested in the query.

Reply:
👍

Reply:
But keep in mind that if I remove that field and a plugin request that it won't work.